encoding.bs

```diff
@@ -3266,6 +3282,7 @@ Aneesh Agrawal,
 Arkadiusz Michalski,
 Asmus Freytag,
 Ben Noordhuis,
+Bnaya Peretz,
```

Comment:
@Bnaya I'm assuming you're okay with this. (Adding you to the Acknowledgments section.)

Reply:
Sure why not:)